src/openms/include/OpenMS/KERNEL/SpectrumHelper.h

```diff
@@ -66,6 +70,34 @@ namespace OpenMS
     return it;
   }
 
+  template <typename PeakContainerT>
+  void slicePeakContainer(PeakContainerT& p, const double pos_start, const double pos_end)
+  {
+    typename PeakContainerT::iterator it_start = p.PosBegin(pos_start);
+    typename PeakContainerT::iterator it_end = p.PosEnd(pos_end);
+    p.erase(it_end, p.end());
+    p.erase(p.begin(), it_start);
+    // Note: only raw peak data is erased
+  }
+
+  template <typename PeakContainerT>
+  void rebaseIntensities(PeakContainerT& p)
```

Comment:
I would recommend the name `reZeroBaseline` as it would indicate that the baseline is moved closer to zero (either up or down depending on if the minimum value is positive or negative).

Reply:
From this comment, it's clear you desire the method to rebase in both cases of negative and positive value